src/__tests__/disableRanges-test.js

```diff
@@ -281,6 +281,29 @@ test("Nesting disabledRanges", t => {
   )
   planCount += 1
 
+  testDisableRanges(
+    `/* stylelint-disable */
+    /* stylelint-enable bar */
+    /* stylelint-enable foo */
+    /* stylelint-disable foo */
+    /* stylelint-enable bar */
+    /* stylelint-enable */`,
```

Comment:
Ok ... Let's do this, then: Instead of adding a test that passes, please cut this test from the PR and open an issue with it, showing that the behavior is buggy in this situation.